instance/models/openedx_instance.py

```diff
@@ -279,6 +279,11 @@ def get_load_balancer_configuration(self):
             # The active appserver doesn't have a public IP address.  This means that the server
```

Comment:
@smarnach Btw, doesn't the call to `ip_address = self.active_appserver.server.public_ip` mean that every change to a single instance will lead to sending a request per instance for all instances to the Nova API? That's a lot of unnecessary requests. It could be something to remember if we find out one day that our calls to the Nova API are being throttled due to the amount that we make.